routes/userRoutes.js

```diff
@@ -66,21 +66,74 @@ router.delete('/:id', passport.authenticate('jwt', { session: false }), (req, re
  * @access  Private
  */
 router.get('/token', passport.authenticate('jwt', { session: false }), (req, res) => {
-	const { user } = req;
-	const payload = {
-		id: user._id,
-		name: user.name,
-		email: user.email,
-		avatar: user.avatar,
-		about: !!user.about && user.about,
-		goals: !!user.goals && user.goals,
-	};
-	const token = createToken(payload, process.env.JWT_SECRET, '1h');
+	User.findById(req.user._id)
+		.populate('friends')
```

Comment:
Maybe we can populate this in passport instead of doing it here? 

Edit: Or maybe not, that would maybe be just overengineering